includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -354,7 +355,8 @@ public static function subscription_pricing_fields() {
 					'class'             => 'wc_input_subscription_intial_price wc_input_subscription_initial_price wc_input_price  short',
 					// translators: %s is a currency symbol / code
 					'label'             => sprintf( __( 'Sign-up fee (%s)', 'woocommerce-subscriptions' ), get_woocommerce_currency_symbol() ),
-					'placeholder'       => _x( 'e.g. 9.90', 'example price', 'woocommerce-subscriptions' ),
+					'placeholder'       => // translators: %s the formatted example price value.
```

Comment:
Similar to an earlier comment, might it be marginally better to escape last? Here we escape the translatable string, but the output of `wc_format_localized_price()` is injected after that, as part of the `sprintf`. 

Escaping last would prevent any accidents that might let the localized price escape the actual HTML attribute.